integration_tests/global_setup/setup.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const crypto = require('crypto');
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+const os = require('os');
+const path = require('path');
+
+const DIR = path.join(os.tmpdir(), '/jest');
+
+module.exports = function() {
+  return new Promise((resolve, reject) => {
```

Comment:
@xfumihiro Everything inside this function in synchronous. So do we really need to wrap this in a promise?
the same applies to tread-down also.

I think this is causing the test to fail.

Reply:
Tests only failed on windows(appveyor) so that can't be the case.
Otherwise all tests should failed also.